test/test_subscriber.cpp

```diff
@@ -101,6 +166,7 @@ TEST(Subscriber, subUnsubSub)
   ASSERT_GT(h.count_, 0);
 }
 
+
```

Comment:
nitpick: Minimize use of vertical whitespace, see: https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace